exercises/shared-api-gateway/ITOps.EndpointConfig/EndpointConfigurationExtensions.cs

```diff
@@ -18,7 +16,7 @@ public static class EndpointConfigurationExtensions
             string connectionString,
             Action<RoutingSettings<MsmqTransport>> configureRouting)
         {
-            NServiceBus.Logging.LogManager.Use<DefaultFactory>();
+            LogManager.Use<DefaultFactory>();
```

Comment:
Is this even necessary?

Reply:
I've no idea.

@Particular/logging-maintainers any idea what this is for? Completely redundant?

I guess I'll just try removing it and if I still get logs, all good!

Reply:
Removed, and logs still appeared.